http/src/test/resources/reference.conf

```diff
@@ -93,6 +93,11 @@ sam {
 
 google {
   leoServiceAccountJsonFile = "fakePath"
+  groups {
+    // TODO Change the default values
```

Comment:
This will eventually need to be in firecloud-develop I think because we need to template environment-specific values

Reply:
Is `TODO` still valid?